topdown/crypto_test.go

```diff
@@ -124,3 +124,43 @@ func TestCryptoSha256(t *testing.T) {
 	}
 
 }
+
+func TestCryptoX509ParseCertificateRequest(t *testing.T) {
+	rule := `
+		p[x] {
+			parsed := crypto.x509.parse_certificate_request(cert)
+			x := parsed.Subject.CommonName
```

Comment:
Instead of only testing for a single value can we do something like:

```rego
p = x {
    x := crypto.x509.parse_certificate_request(cert)
}
```
And then validate the entire payload?

Because we are doing a marshal/unmarshal to round trip from the go library to rego value I think there is value in validating the full object in the tests to ensure nothing weird is being returned.

Reply:
Done